prime-router/metadata/tables/LIVD-SARS-CoV-2-2021-10-27.csv

```diff
@@ -0,0 +1,4819 @@
+Manufacturer,Model,Vendor Analyte Name,Vendor Specimen Description,Vendor Result Description,Test Performed LOINC Code,Test Performed LOINC Long Name,Test Ordered LOINC Code,Test Ordered LOINC Long Name,Vendor Comment,Vendor Analyte Code,Vendor Reference ID,Testkit Name ID,Testkit Name ID Type,Equipment UID,Equipment UID Type,Component,Property,Time,System,Scale,Method,Publication Version ID,LOINC Version ID
```

Comment:
Is this the same as the latest table?  if it is then I suggest not having duplicate files.